src/tests/ftest/launch.py

```diff
@@ -1919,8 +1944,18 @@ def main():
     # Generate certificate files
     generate_certs()
 
+    # Determine the location of the avocado logs for archiving or renaming
+    data = get_output(["avocado", "config"]).strip()
+    avocado_logs_dir = re.findall(r"datadir\.paths\.logs_dir\s+(.*)", data)
+    avocado_logs_dir = os.path.expanduser(avocado_logs_dir[0])
+    print("Avocado logs stored in {}".format(avocado_logs_dir))
+
     # Run all the tests
-    status = run_tests(test_files, tag_filter, args)
+    status = run_tests(test_files, tag_filter, avocado_logs_dir, args)
```

Comment:
This is not needed due to https://github.com/daos-stack/daos/blob/e0744b744866db74b26be9978f4fcd2cf83e55f5/src/tests/ftest/launch.py#L843-L847, which is executed in `run_tests`:
 ```suggestion
    status = run_tests(test_files, tag_filter, args)
```